### DISCUSSION TITLE
Airgapped Enivornment?

### DISCUSSION THREAD

**Discussion by yorkyman** [MISP/MISP#8454] [Q&A]
Can MISP be used in an air gapped enviornment?

**Comment by righel** [comment]
Hello @yorkyman 
Yes, it is possible. You can ingest Events via Feeds, using "Local" option instead of "Network" and referencing a local file.
* See: https://www.circl.lu/doc/misp/managing-feeds/#adding-feeds
You can also take a look at this docker deployment docs for Air-gapped environments:
* https://github.com/NUKIB/misp/blob/main/docs/AIR-GAP.md

Hope this helps.
